spec/features/circle_check_score_update_spec.rb

```diff
@@ -53,7 +46,16 @@
       expect(page).to have_text expected
     end
   end
-
+  context 'when updating with blank field' do
```

Comment:
Generally don't need conjuctions like `when` in an argument to `context` - it reads fine (or better) without them